features/step_definitions/remove_item_steps.rb

```diff
@@ -0,0 +1,11 @@
+Given("I have 1 {string} in my order") do |item|
+  cart = FactoryGirl.create(:cart)
```

Comment:
If you save Cart in an instance variable, you can access it across step definitions. You won't need Cart.last in the next step.

Reply:
Tried it, kept getting `nil` class when I tried to use it in next step.

Reply:
You have to use '@cart' in both step definitions

Reply:
Did that. and just did it again to be sure. it still gives me `nil` class on `@cart` in the 2nd step

Reply:
What if I don't want to be used in both step  definitions?  You should ask for my consent first.

Reply:
sry cart. i will ask for permission next time :)

Reply:
Sorry @cart. 😜 I'll take a look at those step defs what I'm at my computer @magnus-thor Not a blocker for this PR